prac_05/wimbledon.py

```diff
@@ -0,0 +1,46 @@
+CHAMPION_INDEX = 2
```

Comment:
I like that you define the index here 10/10. It removes the weird magic numbers in the code